docs/docs/developer-guides/cosmwasm-cli.md

```diff
@@ -1,35 +1,53 @@
+:::
 
-There are two methods to installing the ccw-vm:
+## Setting up the development environemnt
+
+The process of setting up a development environment for deploying CosmWasm contracts, both a local Picasso netork and on Picasso Rococo, follows the same procedure. There is a distinction in the RPC endpoint mentioned in the CLI commands to upload, instantiate and execute contracts. To interact with a local Picasso network, you will utilize `http://127.0.0.1:32200` whereas to deploy on Picasso Rococo, you will employ `wss://picasso-rococo-rpc-lb.composablenodes.tech:<insert port>`. Additionally, please note that the "-n alice" sudo key will be substituted with your seed phrase when entering the commands.
+
+Nix is a requirement to set up and start a local development environment with Composable's code. We recommend using the [Zero-to-Nix installer](https://zero-to-nix.com/start/install) to install Nix. Refer to our [docs](../nix.md) for more information.
+
+### Installing `ccw`
+
+There are two methods to installing the `ccw-vm`:
 
 1. At first, clone the [Composable moonorepo](https://github.com/ComposableFi/composable):
 
 ```
 git clone https://github.com/ComposableFi/composable
 ```
 
-Then run the following command to install the ccw binary:
+Then run the following command to install the `ccw` binary:
 
 ```
 cargo install --path ./composable/code/parachain/frame/cosmwasm/cli
 ```
 
-2. An alternative method to run the ccw-vm is by running the following command which requires Nix. Nix is required to start the development environment to deploy contracts on Picasso rococo:
+2. An alternative method to run the `ccw-vm` is by running the following command which utilizes Nix. Nix is a package manager used for Composable software and required to start the development environment to deploy contracts on Picasso Rococo:
 
 ```
-nix run composable#ccw
+nix run profile install composable#ccw
 ```
+### Setting up the DevNet
+
+To run a local network with Alice sudo key and start the development environment, run the following commands:
 
-## Installing `nix` 
+```
+cd ./composable
+```
 
-We are using Nix to set up and start our local development environment. So check out our 
-[Nix installation page](https://docs.composable.finance/nix/install) to install Nix.
+```
+nix develop
+```
 
-Then start the development environment by running:
+:::info Nix flags
+If this is your first time using Nix and running on a non-NixOS, ensure you include the correct flags after your Nix commands as outlined [here](https://docs.composable.finance/nix/install#using-flags).
```

Comment:
with zero-to-nix installer, ther only flag from `--extra-experimental-features nix-command --extra-experimental-features flakes --no-sandbox` is `--no-sandbox`. all other flags not needed.

btw I doubt `--no-sandbox` is needed too. not tested so.